crates/wasmtime/src/trap.rs

```diff
@@ -88,6 +88,9 @@ pub enum TrapCode {
     /// Execution has potentially run too long and may be interrupted.
     Interrupt,
 
+    /// Okay why is this defined three times i am losing my mind
```

Comment:
lol not the best situation, yes, but this is the one that is generally public-facing rather than an implementation detail, so it should really have Proper(tm) docs with punctuation and a helpful description and all that.

Reply:
Oh gosh I'm sorry haha, I guess this somehow made it through the "rebase and sanitize 12am thoughts" process